locations/spiders/toysrus.py

```diff
@@ -0,0 +1,73 @@
+import scrapy
+from scrapy.spiders import BaseSpider
+from locations.items import GeojsonPointItem
+import json
+import re
+from io import StringIO
+from scrapy.http import HtmlResponse
+
+default_headers = {
+    "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) "
+                  "Chrome/63.0.3239.84 Safari/537.36"}
+
+def get_hours(hours_obj):
+    out_hours = []
+    for day, hours in hours_obj.items():
+        if hours == "open24":
+            out_hours.append("{}: 24/7".format(day))
+        elif type(hours) is list:
+            ranges = ["{}-{}".format(x["open"], x["close"]) for x in hours]
+            out_hours.append("{}: {}".format(day, ",".join(ranges)))
+    return "; ".join(out_hours)
+
+class ToysRUsSpider(scrapy.Spider):
+    name = "toysrus"
+    download_timeout = 60
```

Comment:
Please don't change the timeout. The timeout is only there for the pull request build so that it gives results back quickly.